src/routes/v1/versions.rs

```diff
@@ -242,44 +233,6 @@ pub async fn get_version_from_hash(
     }
 }
 
-#[derive(Serialize, Deserialize)]
-pub struct DownloadRedirect {
-    pub url: String,
-}
-
-// under /api/v1/version_file/{hash}/download
-#[allow(clippy::await_holding_refcell_ref)]
-#[get("{version_id}/download")]
-pub async fn download_version(
-    info: web::Path<(String,)>,
-    pool: web::Data<PgPool>,
-    algorithm: web::Query<Algorithm>,
-) -> Result<HttpResponse, ApiError> {
-    let hash = info.into_inner().0;
-
-    let result = sqlx::query!(
-        "
-        SELECT f.url url, f.id id, f.version_id version_id, v.mod_id mod_id FROM hashes h
-        INNER JOIN files f ON h.file_id = f.id
-        INNER JOIN versions v ON v.id = f.version_id
-        WHERE h.algorithm = $2 AND h.hash = $1
-        ",
-        hash.as_bytes(),
-        algorithm.algorithm
-    )
-    .fetch_optional(&**pool)
-    .await
-    .map_err(|e| ApiError::Database(e.into()))?;
-
-    if let Some(id) = result {
-        Ok(HttpResponse::TemporaryRedirect()
-            .append_header(("Location", &*id.url))
-            .json(DownloadRedirect { url: id.url }))
-    } else {
-        Ok(HttpResponse::NotFound().body(""))
-    }
-}
-
```

Comment:
this shouldn't be removed

Reply:
I thought that was a private route, but oki

Reply:
Actually this is identical to the equivalent v2 route so I can keep it deleted and just call that

Reply:
Not sure if it was private but iirc people did use it